src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -101,6 +102,10 @@ public boolean hasNoteInIndexer() {
         return m_indexMotor.getForwardLimit().getValueAsDouble() == 0;
     }
 
+    public Trigger hasNote() {
+        return new Trigger(this::hasNoteInIndexer);
+    }
+
```

Comment:
How I'd like to use this trigger:
- onTrue(shooter prep)
- whileTrue(pivot aim)
- debouce(0.5).onFalse(elevator down, pivot to intake position)